app/controllers/documents_controller.rb

```diff
@@ -0,0 +1,69 @@
+class DocumentsController < ApplicationController
+  def index
+      @documents = Document.all.order(created_at: :desc)
+  end
+
+  def new
+    @document = Document.new
+  end
+
+  def create
+    @document = Document.new
+    @document.title = params[:document][:title]
```

Comment:
Consider using strong params! `params.require(:document).permit(:title, :description, :keywords)` could be extracted into a method and then `@document = Document.new(method_name)` would work here.